test/spec/modules/inneractiveBidAdapter_spec.js

```diff
@@ -85,13 +85,11 @@ describe('InneractiveAdapter', function () {
     });
   });
 
-  describe('.createNew()', function () {
-    it('should return an instance of this adapter having a "callBids" method', function () {
-      expect(adapter)
-        .to.be.instanceOf(InneractiveAdapter).and
-        .to.have.property('callBids').and
-        .to.be.a('function');
-    });
+  it('should return an instance of this adapter having a "callBids" method', function () {
```

Comment:
yeah, it only had one test and that test wasn't describing ".creaNew()", so I just removed it.